### ISSUE TITLE
Validate requires TextType but API does not leading to error in response parsing

### ISSUE THREAD

**Issue by dannellyz** [aws-samples/amazon-textract-textractor#201]
The Textractor validation.py has "TextType" as a required block field when the documentation says its options. This causes and error when getting API responses that do not have this field. 

```
TextType
The kind of text that Amazon Textract has detected. Can check for handwritten text and printed text.

Type: String

Valid Values: HANDWRITING | PRINTED

Required: No
```
https://docs.aws.amazon.com/textract/latest/dg/API_Block.html

https://github.com/aws-samples/amazon-textract-textractor/blob/25ac0eeca999446f96f3163c0d099b601d44959c/textractor/validate/validation.py#L72

**Comment by dannellyz** [comment]
Will this be pushed to the Lambda Layers Action?

**Comment by dannellyz** [comment]
Fixed by #203
